logging_config.json

```diff
@@ -0,0 +1,31 @@
+{
+  "version": 1,
+  "formatters": {
+    "default_formatter": {
+      "format": "%(asctime)s %(name)-12s %(levelname)-8s %(message)s"
+    }
+  },
+  "handlers": {
+    "default_logger": {
+      "class": "logging.StreamHandler",
+      "formatter": "default_formatter"
+    }
+  },
+  "loggers": {
```

Comment:
I would prefer to avoid a per-module definition. 
A simple project level config should be used instead.

Reply:
project level config doesn't work as expected. we couldn't find a better way to define the logger